CryptoTracker/CryptoTracker/Scenes/Detail/DetailViewController.swift

```diff
@@ -7,15 +7,17 @@
 
 import UIKit
 import Charts
+import CoreData
 
 final class DetailViewController: UIViewController {
 
-    @IBOutlet weak var priceLabel: UILabel!
+    @IBOutlet private var simpleViews: [UIView]!
```

Comment:
WeAk

Reply:
vâng ạ

Reply:
'weak' may only be applied to class and class-bound protocol types, not '[UIView]'